src/utils/formUtil.ts

```diff
@@ -127,6 +127,8 @@ export const MODEL_TYPES_OF_KNOWN_COMPONENTS = {
     'editgrid',
     'datatable',
     'dynamicWizard',
+  ],
```

Comment:
This all looks good, but can we document `nestedDataArray` in the comments above?